app/web/features/profile/view/UserOverview.tsx

```diff
@@ -43,25 +48,79 @@ const useStyles = makeStyles((theme) => ({
       marginBottom: theme.spacing(0.5),
     },
   },
+
+  cardActions: {
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "stretch",
+    padding: theme.spacing(0.5),
+    "& > *": {
+      margin: theme.spacing(0.5),
+    },
+    "& > :not(:first-child)": {
+      marginLeft: theme.spacing(0.5),
+    },
+  },
 }));
 
-export default function UserOverview({ children }: PropsWithChildren<unknown>) {
+// @todo: move this into /components and decouple it from features/profile because it's used
+//        from the dashboard as well
+export default function UserOverview({
+  actions,
+  showHostAndMeetAvailability = true,
+}: PropsWithChildren<{
+  actions?: React.ReactNode;
+  showHostAndMeetAvailability?: boolean;
+}>) {
   const classes = useStyles();
   const user = useProfileUser();
 
   return (
     <Card className={classes.card}>
-      <Avatar user={user} grow />
+      <Box maxWidth="75%" mx="auto">
+        <Avatar user={user} grow />
+      </Box>
+
       <div className={classes.wrapper}>
-        <Typography variant="h1" className={classes.intro}>
+        <Typography variant="h1" className={classes.intro} align="center">
```

Comment:
```suggestion
        <Typography variant="h1" className={classes.intro}>
```
Probably nitpick, but I'd add it into the `intro` CSS so everything is in one place rather than having this random styling here

Reply:
Sounds good! Makes sense